lib/src/tar/import.rs

```diff
@@ -115,10 +109,41 @@ fn entry_to_variant<R: std::io::Read, T: StaticVariantType>(
     Ok(v.normal_form())
 }
 
+fn parse_object_entry_path(path: &Utf8Path) -> Result<(&str, &Utf8Path, &str)> {
+    let parentname = path
+        .parent()
+        .map(|p| p.file_name())
+        .flatten()
+        .ok_or_else(|| anyhow!("Invalid path (no parent) {}", path))?;
+    if parentname.len() != 2 {
```

Comment:
It took a me a bit to grasp what is this 2 here.
My understanding is that this is checking for the sharding-prefix of the object checksum as the parent directory (sorry, I don't know if this entity has a proper name), am I reading it right?
A brief comment here would be helpful for future readers.

Reply:
OK, added a comment.